src/web/nextui/build.sh

```diff
@@ -3,6 +3,4 @@
 shx mkdir -p dist/src/web/nextui
 npm run build --prefix src/web/nextui
 
-if [ -z "$NEXT_PUBLIC_PROMPTFOO_BUILD_STANDALONE_SERVER" ]; then
-  shx cp -r src/web/nextui/out/* dist/src/web/nextui
-fi
```

Comment:
we always need this in dist since we're never hosting the NextJS server via next